code/modules/projectiles/projectile/bullets.dm

```diff
@@ -93,7 +93,8 @@
 	if(!ishuman(target))
 		return
 	var/mob/living/carbon/human/H = target
-	if(H.getStaminaLoss() >= 60)
+	// 50 - range gives approximate tile distance from user
+	if((50 - range) <= 5 && H.getStaminaLoss() >= 60)
```

Comment:
50 here is the initial range of the rubber pellet (that comes from projectile) right?

Reply:
Yes, all projectiles start out with a range of 50 and tick down by 1 each Range() call. I have no idea if there's a better way to do this, but this is all I could find sorting through the code.

Reply:
You could use "initial(range)" instead, in the future the range could be changed and this would be very easy to miss.

Reply:
Ah, that's a good idea!